Tests/PointFreeTests/BlogTests.swift

```diff
@@ -0,0 +1,107 @@
+import Either
+import Html
+import HtmlPrettyPrint
+import HttpPipeline
+@testable import PointFree
+import PointFreeTestSupport
+import Prelude
+import Optics
+import SnapshotTesting
+import XCTest
+#if !os(Linux)
+import WebKit
+#endif
+
+func withBasicAuth(_ req: URLRequest) -> URLRequest {
```

Comment:
@stephencelis  do you have any ideas of how to bake this into the `request(to:session:)` helper you created? I did this ad hoc for now, but it'd be useful from playgrounds too.

Reply:
oh nice, yeah. i'd prob just add a `basicAuth: Bool` that enables/disables it. maybe `true` by default?

Reply:
we should make some composable setters like `setHeader` :grin: